src/util/merkleblock.rs

```diff
@@ -532,81 +532,94 @@ mod tests {
     use crate::util::merkleblock::{MerkleBlock, PartialMerkleTree};
     use crate::Block;
 
-    #[test]
-    fn pmt_tests() {
+    /// accepts `pmt_test_$num`
+    fn pmt_test_from_name(name: &str) {
+        pmt_test(name[9..].parse().unwrap())
+    }
+
+    macro_rules! pmt_tests {
+    ($($name:ident),* $(,)?) => {
+         $(
+        #[test]
+        fn $name() {
+            pmt_test_from_name(stringify!($name));
+        }
+         )*
+    }
+}
+    pmt_tests!(pmt_test_1, pmt_test_4, pmt_test_7, pmt_test_17, pmt_test_56, pmt_test_100,
+        pmt_test_127, pmt_test_256, pmt_test_312, pmt_test_514, pmt_test_1000, pmt_test_4095);
```

Comment:
question: `pmt_test_4095` (2^12 - 1) is the more expensive, would `pmt_test_1023` (2^10 - 1) accomplish similar purpose?